prive/tests/data/texas-recoding.R

```diff
@@ -0,0 +1,51 @@
+## Recoding of the "Texas" dataset
```

Comment:
I've never used R. Is this something we need in our codebase? It seems more that once we've recoded texas once, we can just use the new csv and forget about this script?

Reply:
Sure ... Maybe I can put this somewhere outside of `prive/`? It's there because the texas dataset came from an external source and for certain at some point we'll want to remember what we did to it!

Reply:
What about `data/original/`?

Reply:
Yeah, I'd be happy with that. My understanding is that the currently encoded Texas dataset is only 999 rows anyway, and we'll need the R script for the full dataset if we want to use that for anything.